packages/fxa-settings/src/models/_mocks.tsx

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { InMemoryCache, ApolloClient, ApolloProvider } from '@apollo/client';
+import { MockedResponse } from '@apollo/client/testing';
+import { Account } from '.';
+import { GET_INITIAL_STATE } from '../components/App';
+import { deepMerge } from '../lib/utilities';
+
+export const MOCK_ACCOUNT: Account = {
+  uid: 'abc123',
+  displayName: 'John Dope',
+  avatarUrl: 'http://avatars.com/y2k',
+  accountCreated: 123456789,
+  passwordCreated: 123456789,
+  recoveryKey: true,
+  attachedClients: [],
+  subscriptions: [],
+  primaryEmail: {
+    email: 'johndope@example.com',
+    isPrimary: true,
+    verified: true,
+  },
+  emails: [
+    {
+      email: 'johndope@example.com',
+      isPrimary: true,
+      verified: true,
+    },
+  ],
+  totp: {
+    exists: true,
+    verified: true,
+  },
+};
+
+export const MOCK_GET_ACCOUNT: MockedResponse = {
+  request: {
+    query: GET_INITIAL_STATE,
+  },
+  result: {
+    data: {
+      account: {
+        ...MOCK_ACCOUNT,
+      },
+    },
+  },
+};
+
+export interface MockedProps {
+  account?: Hash<any>;
+  verified?: boolean;
+  childProps?: object;
+  children?: React.ReactElement;
+}
+export interface MockedState {
+  client: ApolloClient<any>;
+}
+
+/**
+ * Create an InMemoryCache using MOCK_ACCOUNT and optional overrides
+ */
+export function createCache({
+  account = {},
+  verified = true,
+}: MockedProps = {}) {
+  const cache = new InMemoryCache();
+  cache.writeQuery({
+    query: GET_INITIAL_STATE,
+    data: {
+      account: deepMerge({}, MOCK_ACCOUNT, account),
+      session: {
+        verified,
+      },
+    },
+  });
+  return cache;
+}
+
+/**
+ * MockedCache is a sugary sweet version of MockedProvider.
```

Comment:
I'm fine with leaving this here but since it's so lengthy it could also go in the README with a link to the section in the docs here.